workshop/testSelenium.rb

```diff
@@ -0,0 +1,40 @@
+require "test/unit"
+require_relative './selenium.rb'
+
+class SeleniumFrameworkTest < Test::Unit::TestCase
```

Comment:
For the unit tests, you need to create a mock html file to test your framework functions against.
We should not test against the real website.

Reply:
Add negative test cases too

Reply:
Make changes to all unit tests accordingly